src/react-styleguide/src/components/Inputs/Input.js

```diff
@@ -0,0 +1,220 @@
+  // TODO: Map strength value to appropriate width
+  const width = `w-${strength}`
+  const sWord = 'Good'
+
+  return (
+    <div>
+      <h5 className='f5 normal ma0 mb3'>
+        Password Strength: <strong className='fw-5'>{sWord}</strong>
+      </h5>  
+      <div className='w-100 mb3 br-pill overflow-hidden bg-light-gray'>
+        <div className={`bg-blue pa1 ${width}`}></div>
+      </div>
+    </div>
+  )
+}
+PassStrength.propTypes = {
+  strength: React.PropTypes.number.isRequired
+}
+
+export const PasswordTooltip = () => {
+    const passMsg = (
+      <div className='f6 lh-copy'>
+        Use at least 8 characters. Don’t use a password from another site, or something too obvious
+        like your pet’s name. <a href="#" className='orange mh1'>Why?</a>
+      </div>
+    )
+
+  return (
+    <Tooltip>
+      <PassStrength strength={60} />
+      {passMsg}
+    </Tooltip>
+  )
+}
+
+/**
+* Input Examples for Style Guide 
+*/
+export const InputExamples = () => {
+  const Header = ({title}) => (
+    <span className='db f4 fw5 lh-copy mb2'>{title}</span>
+  )
+
+  const exampleLayoutClasses = 'w-100 w-50-m w-33-l pr3-m pl4-l mt2'
+  
+  return(
+    <div className='flex flex-row justify-center flex-wrap items-center'>
+
+      <div className={exampleLayoutClasses}>
+        <Header title='Empty Field' />
+        <Input />
+      </div>
+
+      <div className={exampleLayoutClasses}>
+        <Header title='Field with Placeholder' />
+        <Input placeholder='Placeholder' />
+      </div>
+
+      <div className={exampleLayoutClasses}>
+        <Header title='Disabled Field' />
+        <Input value='Egghead' disabled />
+      </div>
+
+      <div className={exampleLayoutClasses}>
+        <Header title='Error*' />
+        <Input value='Egghead' required error
+          errMsg='Password must contain at least 8 characters.'
+          icon={<XIcon />}
+          value='With Error'
+        />
+      </div>
+
+      <div className={exampleLayoutClasses}>
+        <Header title='Success*' />
+        <Input value='Egghead' required
+          icon={<SuccessIcon />}
+          value='password'
+          type='password'
+        />
+        <PasswordTooltip />
+      </div>
+
+      <div className={exampleLayoutClasses}>
+        <Header title='Icons' />
+        <IconExamples />
+      </div> 
+
+    </div>
+  )
+}
+
+/**
+ * Example Code for above Input Examples
+ * (Ugly formatting, but looks right when rendered)
+ */
+export const InputExampleCode = [
+  `<Input />`,
+  `<Input placeholder='Placeholder' />`, 
+  `<Input value='Egghead' disabled />`,
+  `<Input value='Egghead' required error
+  errMsg='Password must contain at least 8 characters.'
+  icon={<XIcon />}
+  value='With Error'
+/>`,
+ `<Input value='Egghead' required
+  icon={<SuccessIcon />}
+  value='password'
+  type='password'
+/>`,
+ `<PasswordTooltip />`,
+`<SuccessIcon /> <PlusIcon /> <MinusIcon />
+<XIcon /> <BangIcon /> <QuestionIcon />`
+]
```

Comment:
If you want, you could use an npm react markdown library to use markdown for code examples. Might be easier.